### DISCUSSION TITLE
New Source: Splunk

### DISCUSSION THREAD

**Discussion by myacoub91** [airbytehq/airbyte#32674] [New Connector Request]
## Tell us about the new integration you’d like to have
**Which source and which destination? Which frequency?**
Splunk index as a data Source and BigQuery Table as a destination 

## Describe the context around this new integration
**Which team in your company wants this integration, what for? This helps us understand the use case.**
We are currently building pipelines to send data from Splunk index to BigQuery to enable Advanced Cloud Analytics yet there is a huge problem when it comes to SYNC between Source and Destination

## Describe the alternative you are considering or using
**What are you considering doing if you don’t have this integration through Airbyte?**
We are sending data using apache Nifi for sending the data, yet Apache Nifi is blind when it comes to the SYNC option between source and destination here
There is a tool called Data Visually that support that kind of integration , But I don't think it is capable of ensuring that both Source and Destination are in sync
![Screen Shot 2020-09-27 at 12 07 17 AM](https://user-images.githubusercontent.com/56601613/94351163-78df0000-0056-11eb-9de8-ce106acd2bf6.png)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368168312798) by [Unito](https://www.unito.io)

**Comment by sherifnada** [comment]
Hi @myacoub91, thanks for your request! We currently support BigQuery as a destination but do not yet support Splunk as a source, but it is on the roadmap.

**Comment by myacoub91** [comment]
Thanks Sherif, Please let me know once we can work on this , I can support from Splunk perspective as well

[a comment by an automated account is not shown]
